src/index.html

```diff
@@ -17,6 +17,58 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__icon">
+        <a href="#">
+          <img
+            src="/src/images/logo.png"
+            alt="Moyo Icon"
+            width="40"
+            height="40"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list_item">
+            <a
+              class="is-active"
+              href="#"
+            >
+              Apple
```

Comment:
Keep your attributes correctly formatted. If the HTML element has long attribute values or a number of attributes is more than 2, start each one, including the first, on a new line with 2-space indentation related to the tag. Also, the `is-active` class should be on the `class` attribute, not as a standalone attribute.